apollo/handlers/user.py

```diff
@@ -28,11 +30,36 @@ def post_user(user_data: CreateUserSchema,
     data['password_hash'], data['password_salt'] = hash_plaintext(
         user_data.password)
     data.pop('password')
-
     user, _ = save(session, User(**data))
     return get_user_by_id(session, user.id)
 
 
+@router.put('/user/{user_id}', status_code=200,
+            response_model=UserSchema, permission='user.put')
+def put_user(user_id, user_data: UpdateUserSchema, request: Request,
+             session: Session = Depends(get_session)):
+    if (user_id != str(request.current_user.id)):
```

Comment:
I’d opt to not make it generic and expose `user/me` or something which will be easier to document,

Reply:
What do you mean?

Reply:
Make the path `user/me` or `user/self` or whatever instead of `user/{id}` if we're only allowing updates to the current user. It'd be easier for someone reading the docs to understand they can only update themselves.

Reply:
ah op die bike